test/language.e2e-spec.ts

```diff
@@ -263,6 +262,7 @@ describe('Language e2e', () => {
   });
 
   it('should throw error if trying to set hasExternalFirstScripture=true while language has engagements that have firstScripture=true', async () => {
+    await loginAsAdmin(app);
```

Comment:
Is it possible to just give the current user relevant permissions instead of this? 
This makes all subsequent test cases be ran as the admin user.